tutorials/python/python-type-driven-development/tutorial.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Type-Driven Development in Python
+published: 2019-08-01
+tags: Python, typing, types, tdd
+language: python
+author-name: Carlos Villavicencio
+twitter-profile: po5i
+github-profile: po5i
+description: Learn how to improve the software quality and readability in our Python code by using the built-in typing system and complementing with other software development processes like type-driven development and test-driven development.
```

Comment:
instead of "and complementing with other" it may read better to say, "by using the built-in typing system as it complements other"...